### PR TITLE
Py3 for boilerplate check, control flow docs + trivial tweak to code

### PR DESCRIPTION
**Pull request by SgtCoDFish**
Raising as a separate PR to avoid complicating the main signing PRs when they come.

### CODE REVIEW

**Comment by SgtCoDFish** [comment]
/test pull-cert-manager-release-verify

(should be fixed by https://github.com/jetstack/testing/pull/578)

File: cmd/cmrel/cmd/publish.go [hunk 1]
```diff
@@ -40,10 +40,7 @@ Helm charts, generated static manifests and create a release tag on GitHub.
 
 It can only be run by specifying a previously staged build.
 `
-)
-
-var (
-	publishExample = fmt.Sprintf(``)
+	publishExample = ""
 )
```
Comment (SgtCoDFish) [inline cmd/cmrel/cmd/publish.go:44]: note: the diff is slightly unhelpful here; this changes publishExample into a constant

File: hack/verify_boilerplate.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (SgtCoDFish) [inline hack/verify_boilerplate.py:1]: note: both py2 and py3 are installed in the golang-dind docker image we run in

**Review by jakexks — APPROVED** [review]
/lgtm

**Pull request merged**